ui/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
```

Comment:
Since the FE is built on [Vite](https://vitejs.dev), this will actually be http://localhost:5173/.

Reply:
This is the default readme for the react demo. All this code will change with the real UI code, including this file. I'm gonna remove it because nothing written  here is relevant.

BTW: Vite is still not installed. I left this part for FE team because you know much better than me how to use it.

Reply:
Yeah, I am removing this entirely for now in favour of one README in root.